staff/views/users.py

```diff
@@ -42,15 +42,15 @@ def get_context_data(self, **kwargs):
         orgs = [
             {
                 "name": m.org.name,
-                "roles": [r.display_name for r in m.roles],
+                "roles": sorted(r.display_name for r in m.roles),
```

Comment:
A note for myself, because I'm still to some extent trying to remember Python that I used to know.

I wasn't quite sure how this change worked in the sense:

```
>>> sorted(x for x in range(3))
[0, 1, 2]
>>> x for x in range(3)
  File "<stdin>", line 1
    x for x in range(3)
      ^
SyntaxError: invalid syntax
```

Why don't I need parentheses inside the call, but do outside of a call?

> 6.2.8. Generator expressions
> …
> The parentheses can be omitted on calls with only one argument.

https://docs.python.org/3/reference/expressions.html#generator-expressions